rest/route/annotations.go

```diff
@@ -247,12 +247,22 @@ func (h *annotationByTaskPutHandler) Factory() gimlet.RouteHandler {
 func (h *annotationByTaskPutHandler) Parse(ctx context.Context, r *http.Request) error {
 	var err error
 	h.taskId = gimlet.GetVars(r)["task_id"]
+	taskExecutionsAsString := gimlet.GetVars(r)["task_execution"]
```

Comment:
to keep this consistent with other parse functions, we can just call this `execution`.